tests/test_ontology_from_kg.py

```diff
@@ -0,0 +1,173 @@
+
+import logging
+import unittest
+from json import loads
+from falkordb import FalkorDB
+from dotenv import load_dotenv
+from graphrag_sdk.entity import Entity
+from graphrag_sdk.ontology import Ontology
+from graphrag_sdk.relation import Relation
+from graphrag_sdk.attribute import Attribute, AttributeType
+from graphrag_sdk.models.gemini import GeminiGenerativeModel
+from graphrag_sdk import KnowledgeGraph, KnowledgeGraphModelConfig
+
+load_dotenv()
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+
+class TestOntologyFromKG(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+
+        cls.ontology = Ontology()
```

Comment:
I don't think the `ontology` should be part of the actual `class` definition.
I think there's an excessive use of `classmethod` in this test file.
I think there's a different convention for running `setUp` and `tearDown` functions for each instantiation of a test class.